cicd/jenkins-basic/docker/contrib/jenkins/configuration/plugins.txt

```diff
@@ -22,85 +22,85 @@ handy-uri-templates-2-api:2.1.7-1.0
 jquery-detached:1.2.1
 mailer:1.23
 momentjs:1.1.1
-monitoring:1.77.0
-pipeline-model-api:1.3.8
+monitoring:1.78.0
+pipeline-model-api:1.3.9
 plain-credentials:1.5
 pubsub-light:1.12
-scm-api:2.4.1
+scm-api:2.6.3
 simple-theme-plugin:0.5.1
-sse-gateway:1.17
-ssh-credentials:1.16
-token-macro:2.7
+sse-gateway:1.18
+ssh-credentials:1.17.1
+token-macro:2.8
 variant:1.2
-workflow-api:2.34
-workflow-basic-steps:2.15
-workflow-scm-step:2.7
+workflow-api:2.35
+workflow-basic-steps:2.18
+workflow-scm-step:2.9
 workflow-support:3.3
 ace-editor:1.1
 ansicolor:0.6.2
 authentication-tokens:1.3
-blueocean-i18n:1.16.0
-blueocean-jwt:1.16.0
-blueocean-web:1.16.0
-cloudbees-folder:6.8
-credentials-binding:1.18
-docker-commons:1.14
+blueocean-i18n:1.17.0
+blueocean-jwt:1.17.0
+blueocean-web:1.17.0
+cloudbees-folder:6.9
+credentials-binding:1.19
+docker-commons:1.15
 groovy:2.2
 jsch:0.1.55
-junit:1.27
+junit:1.28
 kubernetes-credentials:0.4.0
 matrix-auth:2.4.2
 matrix-project:1.14
-mercurial:2.6
-openshift-login:1.0.16
+mercurial:2.7
+openshift-login:1.0.19
 pipeline-build-step:2.9
 pipeline-input-step:2.10
 pipeline-milestone-step:1.3.1
 pipeline-stage-step:2.3
-pipeline-stage-tags-metadata:1.3.8
+pipeline-stage-tags-metadata:1.3.9
 rebuild:1.31
-timestamper:1.9
-workflow-cps:2.68
-workflow-durable-task-step:2.30
-workflow-job:2.32
-blueocean-config:1.16.0
-blueocean-dashboard:1.16.0
-blueocean-personalization:1.16.0
-branch-api:2.4.0
+timestamper:1.10
+workflow-cps:2.72
```

Comment:
This is causing some issues...[Pipeline CPS method mismatch](https://github.com/openshift/jenkins-client-plugin/issues/283). 
Although our pipeline still works, it is littered with:
`expected to call com.openshift.jenkins.plugins.OpenShiftDSL.dieIfWithout but wound up catching com.openshift.jenkins.plugins.OpenShiftDSL.contextContains; see: https://jenkins.io/redirect/pipeline-cps-method-mismatches/`